static/js/zamboni/static_theme.js

```diff
@@ -224,8 +226,12 @@ $(document).ready(function() {
                     $wizard.find('#submit-describe').submit();
                 } else {
                     data.validation.messages.forEach(function(message) {
-                       $('.errorlist.validator').append($('<li>', {'html': message.message}));
-                       console.error(message);
+                       if (headerImageError) {
+                        $('.header-image-error').append($('<li>', {'html': message.message}));
+                       } else {
+                        $('.theme-name-error').append($('<li>', {'html': message.message}));
+                       }
+                       console.error(data);
```

Comment:
this is changed from `console.error(message)` - do we need the extra information from the data object?

Reply:
No we don't. I was using it for debugging purposes. Seems like I forgot to change it back.